bitnami/memcached/values-production.yaml

```diff
@@ -112,6 +120,22 @@ nodeSelector: {}
 ##
 tolerations: {}
 
+## Persistence - used for dumping and restoring states between recreations
+## Ref: https://github.com/memcached/memcached/wiki/WarmRestart
+persistence:
+  enabled: false
+  storageClassName: ""
+  # requests:
+  #   storage: 5Gi
+
+## Args for running memcached
+## Ref: https://github.com/memcached/memcached/wiki/ConfiguringServer
+command:
+  - memcached
+  # - -m <maxMemoryLimit>
+  # - -I <maxItemSize>
+  # - -vv
```

Comment:
We're using some scripts (`entypoint.sh` and `run.sh`) to start Memcached on this image. See:

- https://github.com/bitnami/bitnami-docker-memcached/blob/master/1/debian-9/Dockerfile#L25
- https://github.com/bitnami/bitnami-docker-memcached/blob/master/1/debian-9/rootfs/entrypoint.sh
- https://github.com/bitnami/bitnami-docker-memcached/blob/master/1/debian-9/rootfs/run.sh

Basically, the `entypoint.sh` is designed to do some setup (see https://github.com/bitnami/bitnami-docker-memcached/blob/master/1/debian-9/rootfs/setup.sh) when the command is "run.sh". This setup is based on some functions defined at  https://github.com/bitnami/bitnami-docker-memcached/blob/master/1/debian-9/rootfs/libmemcached.sh

We should maintain using these scripts since they provide us a lot of flexibility to configure Memcached.

Finally, if you check the line below, you'll notice that we're already appending every extra parameter to the list of parameters used to start memcached:

https://github.com/bitnami/bitnami-docker-memcached/blob/master/1/debian-9/rootfs/run.sh#L26

Reply:
The approach is a bit convoluted - but let's stick to it for consistency. The current chart has no way of adding env-vars, for instance to configure MEMCACHED_CACHE_SIZE. Also the env-vars are not documented - or am I missing something here?